ddtrace/debugging/_probe/model.py

```diff
@@ -156,22 +156,26 @@ def location(self):
         return (maybe_stringify(self.resolved_source_file), self.line)
 
 
-class ProbeEvaluateTimingForMethod(str, Enum):
+class ProbeEvalTiming(str, Enum):
     DEFAULT = "DEFAULT"
-    ENTER = "ENTER"
```

Comment:
scary...

Reply:
The RFC mentions `ENTRY` as accepted value so I'm aligning with that